### PR TITLE
Dropdown: fix TS error

### PR DESCRIPTION
**Pull request by rosslo**

### CODE REVIEW

File: src/Display/Slider/Slider.tsx [hunk 1]
```diff
@@ -148,7 +148,7 @@ class Slider extends React.Component<Props, State> {
     window.addEventListener('resize', this.setSize);
   }
 
-  componentWillReceiveProps(nextProps: Props) {
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
```
Comment (rosslo) [inline src/Display/Slider/Slider.tsx:151]: somehow we missed this intended change haha https://github.com/glints-dev/glints-aries/pull/269#discussion_r404549173

**Comment by tingtingdisco** [comment]
@rosslo I still don't know why `Pick` makes all the optional properties required, do you think we can use `Partial` to make these optional properties still visible in the file?

```
type PropsFromDropdownHeader = Partial<
  Pick<
    React.ComponentPropsWithoutRef<typeof DropdownHeader>,
    'disabled' | 'showHoverLine' | 'showFullWidth'
  >
>;

type PropsFromDropdownBody = Partial<
  Pick<
    React.ComponentPropsWithoutRef<typeof DropdownBody>,
    'dropDownPlacement' | 'noLineBreak' | 'showFullWidth' | 'showHoverLine'
  >
>;
```

**Review by tingtingdisco — APPROVED** [review]
LGTM, @rosslo we can figure out why Pick makes optional properties required later.

**Comment by rosslo** [comment]
> LGTM, @rosslo we can figure out why Pick makes optional properties required later.

Okay, thanks @tingtingdisco

**Pull request merged**